src/ui/LysisUI.java

```diff
@@ -0,0 +1,257 @@
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.datatransfer.StringSelection;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+
+public class LysisUI extends JFrame
+{
+    /* Window Settings */
+    public static final int WINDOW_WIDTH = 700;
+    public static final int WINDOW_HEIGHT = 270;
+
+    /* Instance Variables */
+    JTextArea output;
+    String path;
+    boolean decompiled;
+
+    public LysisUI()
+    {
+        super("Lysis Decompiler");
+
+        this.path = null;
+        this.decompiled = false;
+
+        /* Window Settings */
+        this.setSize(WINDOW_WIDTH, WINDOW_HEIGHT);
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+
+        /* North Panel */
+        JPanel north = new JPanel();
+        north.setLayout(new FlowLayout());
+        north.setBackground(Color.gray);
+
+        JLabel title = new JLabel("Lysis Decompiler");
+        Font font = new Font("Atlantic Cruise", Font.BOLD, 13);
+        title.setFont(font);
+        title.setForeground(Color.white);
+        north.add(title);
+
+        /* West Panel */
+        JPanel west = new JPanel();
+        west.setLayout(new FlowLayout());
+        west.setPreferredSize(new Dimension(120, 400));
+        west.setBackground(Color.gray);
+
+        JButton loadFile = new JButton("Load File");
+        loadFile.setPreferredSize(new Dimension(120, 27));
+        loadFile.addActionListener(new LoadFileListener());
+
+        JButton saveFile = new JButton("Save File");
+        saveFile.setPreferredSize(new Dimension(120, 27));
+        saveFile.addActionListener(new SaveButtonListener());
+
+        JButton copyText = new JButton("Copy All");
+        copyText.setPreferredSize(new Dimension(120, 27));
+        copyText.addActionListener(new CopyButtonListener());
+
+        JButton decompile = new JButton("Decompile!");
+        decompile.setPreferredSize(new Dimension(120, 27));
+        decompile.addActionListener(new DecompileButtonListener());
+
+        west.add(Box.createRigidArea(new Dimension(120, 10)));
+        west.add(loadFile);
+        west.add(Box.createRigidArea(new Dimension(120, 10)));
+        west.add(saveFile);
+        west.add(Box.createRigidArea(new Dimension(120, 10)));
+        west.add(copyText);
+        west.add(Box.createRigidArea(new Dimension(120, 10)));
+        west.add(decompile);
+
+        /* Center panel */
+        output = new JTextArea();
+        output.setEditable(false);
+        output.setBackground(Color.black);
+        output.setFont(new Font("Consolas", Font.PLAIN, 12));
+        output.setForeground(Color.white);
+
+        /* Add panels to frame */
+        this.add(north, BorderLayout.NORTH);
+        this.add(west, BorderLayout.WEST);
+        this.add(new JScrollPane(output), BorderLayout.CENTER);
+
+        this.setVisible(true);
+        this.setResizable(false);
```

Comment:
That's do-able. I'd just have to change the button configuration on the left such that when the user expands the window the space between the buttons grow with it.